docs/spec/observer/messages.md

```diff
@@ -70,3 +70,20 @@ message MsgUpdateKeygen {
 }
 ```
 
+## MsgAddBlockHeader
+
+Updates permissions. Currently, this is only used to enable/disable the
+inbound transactions.
+
+Only the admin policy account is authorized to broadcast this message.
+
+```proto
+message MsgAddBlockHeader {
+	string creator = 1;
+	int64 chain_id = 2;
+	bytes tx_hash = 3;
```

Comment:
It's better to rename this 'tx_hash' as 'block_hash' if we posted a block's hash. when call below function in evm client.
`ob.zetaClient.PostAddBlockHeader(ob.chain.ChainId, block.Hash().Bytes(), block.Number().Int64(), headerRLP)`